src/libs/OptionsListUtils.js

```diff
@@ -452,12 +453,12 @@ function getSidebarOptions(reports, personalDetails, draftComments, activeReport
  */
 function getHeaderMessage(hasSelectableOptions, hasUserToInvite, searchValue, maxParticipantsReached = false) {
     if (maxParticipantsReached) {
-        return CONST.MESSAGES.MAXIMUM_PARTICIPANTS_REACHED;
+        return translate(ONYXKEYS.PREFERRED_LOCALE, CONST.MESSAGES.MAXIMUM_PARTICIPANTS_REACHED);
```

Comment:
Same comment as in other files, we need to use withLocalize

Reply:
Same as above, cannot use withLocalize outside react components